app/controllers/teachers_controller.rb

```diff
@@ -67,14 +67,12 @@ def update
     @teacher.save!
     @school.save!
     if is_admin?
-      flash[:success] = "Saved #{@teacher.full_name}"
-      redirect_to teachers_path, notice: "Successfully updated information"
+      redirect_to teachers_path, notice: "Saved #{@teacher.full_name}"
       return
     end
     # Resends emails only when teacher (not admin) updates and not validated
     if @teacher.application_status != "Validated"
       TeacherMailer.form_submission(@teacher).deliver_now
```

Comment:
Why did we originally send teals email here? @christopher-hou

Reply:
I asked Michael if he wanted TEALS emails to be resent if teachers update their info and he said yes. I can ask him about this on Wednesday's meeting just to confirm what his current thoughts are